static/src/javascripts/projects/commercial/modules/brazeBanner.js

```diff
@@ -200,7 +199,6 @@ const canShow = async () => {
     }
 
     if (!canShowPreChecks({
-        userIsGuSupporter: shouldNotBeShownSupportMessaging(),
         pageConfig: config.get('page'),
     })) {
         // Currently all active web canvases in Braze target existing supporters,
```

Comment:
We should probably remove this comment now, since it's no longer accurate.

Reply:
Removed!